stories/spectra/nmr.stories.tsx

```diff
@@ -49,7 +49,6 @@ export function StackSpectra() {
       xAxis="x"
       yAxis="y"
       lineStyle={{ stroke: color }}
-      yShift={`${i * -7}`}
```

Comment:
can we keep the shift and padding as before?

Reply:
Yes no probleme, 
i just thought that the correct graph is without shift

Reply:
fixed